kukkuu/settings.py

```diff
@@ -42,7 +42,7 @@
     TOKEN_AUTH_ACCEPTED_SCOPE_PREFIX=(str, "kukkuu"),
     TOKEN_AUTH_REQUIRE_SCOPE_PREFIX=(bool, True),
     TOKEN_AUTH_AUTHSERVER_URL=(str, ""),
-    ILMOITIN_QUEUE_NOTIFICATIONS=(str, True),
+    ILMOITIN_QUEUE_NOTIFICATIONS=(str, False),
```

Comment:
so, for local dev env you want to disable queueing notifications?

Reply:
I think that should be better. Since in local we probably don't use uwsgi but default django runserver to run the app so dev might don't want to setup cronjob in his local machine

Reply:
yes, sure! just checking that this was the goal here 🙂